update.py

```diff
@@ -1009,6 +1005,25 @@ def runSphinxAutoBuild():
         "python misc/sphinx_autobuild_wrapper.py doc output/ --watch misc --watch update.py --watch doc --watch intl --watch Pipenv.lock"
     )
 
+def getTranslationStatus():
+    status = {}
+    for path in Path("doc").rglob("*.rst"):
+        translations = []
+        for locale in Path("locales").glob("*"):
+            if Path(locale, "LC_MESSAGES", f"{str(path.relative_to('doc'))[:-3]}po").exists():
+                translations.append(locale.name)
+
+            status[path] = translations
+
+    return status
+
+def updateTranslationStatusPage():
+    table = [["Site", "Translations"]]
+    for path, translations in getTranslationStatus().items():
+        if translations:
+            table += [[str(path.relative_to("doc")).replace("\\", "/"), ", ".join(translations)]]
+
+    putTextFileContents(Path("doc", "translation-status.rst"), makeTable(table))
```

Comment:
I suspect, this is missing a template file that removes it from the index or something, or else sphinx should complain about the orphan page. I will add a template then.